src/v2/Apps/Artist/Routes/Overview/Components/ArtistConsignButton.tsx

```diff
@@ -80,11 +80,24 @@ export const ArtistConsignButton: React.FC<ArtistConsignButtonProps> = ({
       </RouterLink>
 
       <Media greaterThanOrEqual="md">
-        <Button variant="secondaryGray">Get started</Button>
+        <Button
+          variant="secondaryGray"
+          // @ts-ignore
+          as={RouterLink}
```

Comment:
@dzucconi - can we type `as` in Palette as Any and then just forward that down with a ts-ignore?

Reply:
Yeah; `Button` is an easy target; you can't exactly just type `as` since it wouldn't ignore subsequent props like `to`

I'll do something like this: https://github.com/artsy/palette/blob/56f21896941be8c28e63a5c17566fbd610cf0e4d/packages/palette/src/elements/BaseTabs/BaseTab.tsx#L16-L23